config/example_test.go

```diff
@@ -26,69 +26,24 @@ import (
 	"github.com/uber/jaeger-lib/metrics"
 
 	"github.com/uber/jaeger-client-go"
-	. "github.com/uber/jaeger-client-go/config"
-	jaeger_log "github.com/uber/jaeger-client-go/log"
+	jaegercfg "github.com/uber/jaeger-client-go/config"
+	jaegerlog "github.com/uber/jaeger-client-go/log"
 )
 
 func ExampleInitialization() {
-	cfg := Configuration{
-		Disabled: false,
-		Sampler: &SamplerConfig{
+	cfg := jaegercfg.Configuration{
+		Sampler: &jaegercfg.SamplerConfig{
```

Comment:
Or perhaps have two functions, ExampleInitializationForProduction (essentially an empty config) and ExampleInitializationForTesting (can show how to enable span logging).